packages/cms/src/components/editors/vizLookup.js

```diff
@@ -0,0 +1,138 @@
+// Placeholder for any future universal keys
+const sharedMethods = [];
+
+// Create a lookup for some commonly used keys
+const commonKeys = ["groupBy", "x", "y", "value", "colorScale", "label", "subtitle", "sum"];
+const commonMethods = {};
+commonKeys.forEach(key => {
+  commonMethods[key] = {
+    key,
+    display: key,
+    format: "Accessor",
+    required: true
+  };
+});
+// Override some of the labels and types
+commonMethods.groupBy.typeof = "id";
+commonMethods.groupBy.display = "Grouping";
+commonMethods.colorScale.display = "Color Scale";
+commonMethods.value.typeof = "number";
+commonMethods.sum.typeof = "number";
+
+// Create a lookup for some optional keys. Nested properties are handled by dots, e.g. yConfig.title
+const optionalKeys = ["xConfig", "yConfig"];
+optionalKeys.forEach(key => {
+  commonMethods[key] = {
+    key: `${key}.title`,
+    display: `${key.charAt(0).toUpperCase()} Axis Label`,
+    format: "Input",
+    required: false
+  };
+});
+
+// Promote certain keys to appear in the tooltip by default
+const tooltipKeys = ["x", "y", "value", "sum"];
+tooltipKeys.forEach(key => {
+  commonMethods[key].tooltip = true;
+});
+const {groupBy, x, y, value, colorScale, label, subtitle, sum, xConfig, yConfig} = commonMethods;
+
+module.exports = [
+  {
+    name: "Area Plot", // the name to be displayed in UI mode
+    type: "AreaPlot", // the actual d3plus component (the "type" key)
+    methods: [groupBy, x, y, xConfig, yConfig, ...sharedMethods]
+  },
```

Comment:
This one's actually not really used (it's usually always `StackedArea`, so let's remove this for now.